src/main/java/com/github/ChristopheCVB/EliteDangerous/events/fleetcarriers/CarrierModulePackEvent.java

```diff
@@ -0,0 +1,20 @@
+package com.github.ChristopheCVB.EliteDangerous.events.fleetcarriers;
+
+import com.github.ChristopheCVB.EliteDangerous.events.Event;
+
+public class CarrierModulePackEvent extends Event {
```

Comment:
`CarrierModulePackEvent` and `CarrierShipPackEvent` have the same data in it.
Knowing that, it may be smart to create a `public abstract class CarrierPackEvent extends Event` containing the data and the former 2 will extend the last. The `CarrierPackEvent` does not embed a Listener because there is no `CarrierPack` event in the documentation.